package.json

```diff
@@ -63,7 +63,7 @@
     "classnames": "^2.2.5",
     "hogan.js": "^3.0.2",
     "htm": "^3.0.0",
-    "preact": "^10.6.0",
+    "preact": "^10.10.0",
```

Comment:
This fixes a TypeScript overlook to allow `defaultChecked` on `HTMLInputElement`s.